### PR TITLE
Use prepared queries to improve parsing and planning time

### PR DESCRIPTION
**Pull request by Raveline**
Implements #20.

### CODE REVIEW

File: src/Database/PostgreSQL/Consumers/Utils.hs [hunk 1]
```diff
@@ -73,3 +78,6 @@ forkImpl ffork tname m = E.mask $ \release -> do
       E.Handler $ \StopExecution -> return ()
     , E.Handler $ (throwTo parent . ThrownFrom tname)
     ]
+
+hashedName :: Text -> RawSQL () -> QueryName
+hashedName baseName tableName = QueryName . (mappend "consumers")  . pack . show . hash $ unRawSQL tableName <> baseName
```
Comment (arybczak) [inline src/Database/PostgreSQL/Consumers/Utils.hs:83]: IIUC with this you won't know from postgresql logs for which table/operation the query is executed, I doubt @marco44 would like that :slightly_smiling_face: 

I propose to just do `take 63 $ baseName ++ "$" ++ tableName`, it's doubtful there will be two job or consumer tables with the same prefix of length ~50.

Comment (Raveline) [reply src/Database/PostgreSQL/Consumers/Utils.hs:83]: Easy enough to fix.

**Review by arybczak — COMMENTED** [review]
Please bump the version to 2.3.2.0 and add changelog entry.

File: src/Database/PostgreSQL/Consumers/Utils.hs [hunk 2]
```diff
@@ -4,14 +4,18 @@ module Database.PostgreSQL.Consumers.Utils (
...
+  , uniqueName
   ) where
 
 import Control.Concurrent.Lifted
 import Control.Monad.Base
 import Control.Monad.Catch
 import Control.Monad.Trans.Control
+import Database.PostgreSQL.PQTypes.SQL.Raw
+import Database.PostgreSQL.PQTypes.Class
+import Data.Text
```
[the remark this answers is not in the document]
Comment (Raveline) [reply src/Database/PostgreSQL/Consumers/Utils.hs]: Sure. Done.

File: src/Database/PostgreSQL/Consumers/Utils.hs [hunk 3]
```diff
@@ -73,3 +77,6 @@ forkImpl ffork tname m = E.mask $ \release -> do
       E.Handler $ \StopExecution -> return ()
     , E.Handler $ (throwTo parent . ThrownFrom tname)
     ]
+
+uniqueName :: Text -> RawSQL () -> QueryName
```
[the remark this answers is not in the document]
Comment (Raveline) [reply src/Database/PostgreSQL/Consumers/Utils.hs]: Much better name indeed !

**Comment by Raveline** [comment]
Bumped and updated the CHANGELOG. Am I ok to merge ?

**Pull request merged**